src/platform/windows.rs

```diff
@@ -684,10 +712,10 @@ pub fn try_change_desktop() -> bool {
 }
 
 fn share_rdp() -> BOOL {
-    if get_reg("share_rdp") != "true" {
-        FALSE
-    } else {
+    if get_reg("share_rdp") != "false" {
         TRUE
+    } else {
+        FALSE
```

Comment:
Make sure corresponding option in settings UI page also changed, and test please.